packages/core/useFetch/index.ts

```diff
@@ -18,17 +18,17 @@ export interface UseFetchReturn<T> {
   /**
    * The raw response of the fetch response
    */
-  response: Ref<Response | null>
+  response: ShallowRef<Response | null>
 
   /**
    * Any fetch errors that may have occurred
    */
-  error: Ref<any>
+  error: ShallowRef<any>
 
   /**
    * The fetch response body, may either be JSON or text
    */
-  data: Ref<T | null>
+  data: ShallowRef<T | null>
```

Comment:
Not sure. I think `Ref<T>` is already shallow.

Reply:
![image](https://user-images.githubusercontent.com/22525904/155842984-f66517fc-2d44-4c3e-8f36-f7efa41a4d33.png)

in `useFetch` `data` and `response` is using `shallowRef` but the types is `Ref`.

https://github.com/vueuse/vueuse/blob/37a0205ed9668be06f65b51fad1ce36906084c43/packages/core/useFetch/index.ts#L300-L302

in `useAxios` `error` is using `shallowRef`. maybe in `useFetch` `error` should be shallow too.

https://github.com/vueuse/vueuse/blob/f464846a09906433c8dca876ee87c18630d0f33c/packages/integrations/useAxios/index.ts#L91-L96

Reply:
Yeah, it makes sense to have error as `shallowRef`, but there is no need to change the types.